backend/api.js

```diff
@@ -19,15 +11,37 @@ const foundParams = (params, wantedParam) => {
     return false;
 };
```

Comment:
this can be simplified to just be `const foundParams = (params, wantedParam) => Boolean(params) && has(wantedParam, params);`